docs/design/launch_system.md

```diff
@@ -22,15 +23,21 @@ Each folder contains a `launch/` folder where all the launch files should be pla
 
 ### File Naming Convention
 
-All launch files end in `.launch.py`. Furthermore, all launch files specific to the a vehicle platform or orchastrator launch files are prefixed with `art_`.
+All launch files end in `.launch.py`. Furthermore, all launch files specific to a vehicle platform or orchastrator launch files are prefixed with `art_`.
 
 ## `art_launch/`
 
-This is where the main launch file is held: [`art.launch.py`](../../workspace/src/common/launch/art_launch/launch/art.launch.py). This file will do a few things.
+This is where the main launch file is held: [`art.launch.py`](../../workspace/src/common/launch/art_launch/launch/art.launch.py). The only purpose of this file is to define system wide parameters (e.g. `LaunchConfigurations`, `LaunchDescriptions`, etc.). This will allow all orchestrators to know which specific packages to launch, and which to ignore. Following this, ther remainder of the general setup is left to the `art_shared_launch`.
 
-1. It will first define system wide parameters (e.g. `LaunchConfigurations`, `LaunchDescriptions`, etc.).
-2. It will create a [composable node container](https://docs.ros.org/en/galactic/How-To-Guides/Launching-composable-nodes.html).
-3. It will include all other orchestration launch files.
+> [!NOTE]
+> Vehicle-specific launch fies should be named following the above naming convention (`art_<name>_launch`), and should be structured in a similar way to `art_launch`. This means they should only define launch arguments, leaving the process of actually launching modules to the `art_shared_launch`.
+
+## `art_shared_launch/`
+
+This is where the actual modules get launched from. the `art_shared_launch` acomplishes a few things, which will be common across all vehicles.
```

Comment:
accomplishes

Reply:
And what do you mean by the first sentence? I think you mean this is where common configurations are made, right?